web/utils/language.ts

```diff
@@ -10,54 +11,67 @@ export const languages = [
   {
     value: 'en-US',
     name: 'English(United States)',
+    example: 'Hello, Dify!',
   },
   {
     value: 'zh-Hans',
     name: '简体中文',
+    example: '你好，Dify！',
   },
   {
     value: 'pt-BR',
     name: 'Português(Brasil)',
+    example: 'Olá, Dify!',
   },
   {
     value: 'es-ES',
     name: 'Español(España)',
+    example: 'Saluton, Dify!',
   },
   {
     value: 'fr-FR',
     name: 'Français(France)',
+    example: 'Bonjour, dify!',
   },
   {
     value: 'de-DE',
     name: 'Deutsch(Deutschland)',
+    example: 'Hallo, Dify!',
   },
   {
     value: 'ja-JP',
     name: '日本語(日本)',
+    example: 'こんにちは、ディファイ!',
```

Comment:
I think we can keep `Dify` for none English languages and keep them as Capitalized.

Reply:
> I think we can keep `Dify` for none English languages and keep them as Capitalized.

example has been adjusted